zerver/tests/test_digest.py

```diff
@@ -121,7 +52,7 @@ def test_multiple_stream_senders(self,
         with queries_captured() as queries:
             handle_digest_email(othello.id, cutoff)
 
-        self.assertTrue(24 <= len(queries) <= 25)
+        self.assertTrue(21 <= len(queries) <= 221)
```

Comment:
why did this number jump up?

Reply:
Ah, that's a typo. It should be `22`. :see_no_evil: